Library/Formula/bwctl.rb

```diff
@@ -16,15 +16,99 @@ class Bwctl < Formula
   depends_on "iperf3" => :optional
   depends_on "thrulay" => :optional
 
+  # Patch to fix failure to start under launchd. Submitted upstream at
+  # https://code.google.com/p/perfsonar-ps/issues/detail?id=1043
+  patch :DATA
+
   def install
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--mandir=#{man}"
     system "make", "install"
+
+    (buildpath+"bwctld.conf").write <<-EOS.undent
+      allow_unsync
+      log_location
+      peer_port       6001-6200
+      facility        local5
+      test_port       5001-5900
+      iperf_port      5001-5300
+      nuttcp_port     5301-5600
+      owamp_port      5601-5900
+      user            nobody
+      group           nobody
+      EOS
+    (buildpath+"bwctld.keys").write ""
+    (etc+"bwctld").mkpath
+    (etc+"bwctld").install "bwctld.conf" => "bwctld.conf",
+                           "bwctld.keys" => "bwctld.keys",
+                           "conf/bwctld.conf" => "bwctld.conf.default",
+                           "conf/bwctld.limits" => "bwctld.limits"
+
+    (var+"log/bwctld").mkpath
```

Comment:
This should happen automatically with the new `plist`. We have *some* intelligent directory creation in regards to `plist` handling.